azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/PermissionManager.kt

```diff
@@ -111,34 +148,10 @@ internal class PermissionManager(
         store.dispatch(PermissionAction.CameraPermissionIsSet(getCameraPermissionState(activity)))
     }
 
+    fun setPhonePermissionState() {
+        store.dispatch(PermissionAction.PhonePermissionIsSet(getPhonePermissionState(activity)))
+    }
+
     private fun isPermissionGranted(permission: String) =
         ActivityCompat.checkSelfPermission(activity, permission) == PackageManager.PERMISSION_GRANTED
-
-    private fun getPermissionsList(): Array<String> =
```

Comment:
Just wondering why was this removed?

Reply:
IDE said it was unused so I deleted.

We have a list of permissions for audio, and a single permission for camera/phone_state permission requests. I think this is just old dead code.